src/main/java/org/zalando/nakadi/controller/NakadiController.java

```diff
@@ -0,0 +1,74 @@
+package org.zalando.nakadi.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+import org.zalando.nakadi.domain.ItemsWrapper;
+import org.zalando.nakadi.security.Client;
+import org.zalando.nakadi.service.FloodService;
+import org.zalando.nakadi.util.FeatureToggleService;
+
+import static org.zalando.nakadi.security.Client.FULL_ACCESS;
+
+@RestController
+@RequestMapping(value = "/nakadi")
+public class NakadiController {
```

Comment:
>Implementation supports access only for specified oauth uid, so called application id. https://github.com/zalando/nakadi/blob/master/src/main/resources/application.yml#L59
Only the owning team is allowed to request ouath token for that application.

That sounds la Nakadi user (producer/consumer) can't ever use these APIs and only an operator can. If so, :ok: